Source/Scene/Points3DTileContent.js

```diff
@@ -166,11 +168,97 @@ define([
         var pointsLength = view.getUint32(byteOffset, true);
         byteOffset += sizeOfUint32;
 
-        var positionsOffsetInBytes = byteOffset;
-        var positions = new Float32Array(arrayBuffer, positionsOffsetInBytes, pointsLength * 3);
+        var batchTableJSONByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
+        var batchTableBinaryByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
+        var positions = new Float32Array(arrayBuffer, byteOffset, pointsLength * 3);
```

Comment:
I'm surprised we haven't run into this before, but TypedArray constructors expect their byte offset to be aligned to the type. For two of the points tilesets, the byte length of the batch table is odd and this causes problems when creating a `Float32Array` of the positions. Do we want to require byte alignment in the spec?